news/2025/2025-01-31-osumania-7k-world-cup-2025-round-of-16-recap.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title: "osu!mania 7K World Cup 2025: Round of 16 Recap"
+date: 2025-01-31 12:00:00 +0000
+---
+
+With the the tournament drawing near to its halfway point, our contestants' fighting spirits are on the rise!
+
+![](/wiki/Tournaments/MWC/2025_7K/img/mwc7k2025-banner.jpg)
+
+With a steady increase in mappool difficulty, our teams are starting to get accustomed to the pace of the tournament. Competition is steadily on the rise as our teams fight relentlessly for glory or to save themselves from the clutches of elimination. With no further ado, let's see what this week's Round of 16 had in store for us!
+
+Moreover, we got the chance to host an interview with [ChillierPear](https://osu.ppy.sh/users/9501251) in the hopes of offering us some insight on world cup management as a whole, putting into perspective what a massive endeavour it is to host a series of tournaments of this scale. Stay tuned for the interview, or spoil yourself by scrolling all the way down!
+
+As per usual, [SurfChu85](https://osu.ppy.sh/users/4469895) and [yukina meng](https://osu.ppy.sh/users/9919550) have joined forces once again to recap the events in video form for those who'd prefer to watch along instead.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/OsZ0Taiqbr0" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Mappool recap
+
+**[Download the pool here! (77 MB)](https://osu.ppy.sh/beatmaps/packs/P272)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2358013328&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+![Round of 16 mappool table](/wiki/shared/news/2025-01-31-osumania-7k-world-cup-2025-round-of-16-recap/mappool.jpg)
+
+As to be expected, the mappool had a substantial bump in difficulty, but it's probably not going to be apparent by the amazing scores the players have managed to set, as you'll soon see! Nevertheless, this is easier said than done, especially on the very first elimination round. Teams cannot afford to grow complacent, and the pools to come won't make that an easy goal to achieve.
```

Comment:
idk if i would say that getting complacent is anyone's goal? maybe say that it wont be easy to become complacent with the difficulty of the pool ramping up
```suggestion
As to be expected, the mappool had a substantial bump in difficulty, but it's probably not going to be apparent by the amazing scores the players have managed to set, as you'll soon see! Nevertheless, this is easier said than done, especially on the very first elimination round. Teams cannot afford to grow complacent, and the pools to come won't make that an easy goal to achieve.
```

Reply:
```suggestion
As to be expected, the mappool had a substantial bump in difficulty, but it's probably not going to be apparent by the amazing scores the players have managed to set, as you'll soon see! Nevertheless, this is easier said than done, especially on the very first elimination round. Teams cannot afford to grow complacent, because the pools to come won't become any easier.
```